modules/apps/27-interchain-accounts/controller/types/msgs_test.go

```diff
@@ -143,7 +148,15 @@ func TestMsgSendTxValidateBasic(t *testing.T) {
 			Amount:      ibctesting.TestCoins,
 		}
 
-		data, err := icatypes.SerializeCosmosTx(simapp.MakeTestEncodingConfig().Marshaler, []proto.Message{msgBankSend})
+		tempApp := simapp.NewSimApp(log.NewNopLogger(), dbm.NewMemDB(), nil, true, simtestutil.NewAppOptionsWithFlagHome(t.TempDir()))
+		encodingConfig := params.EncodingConfig{
```

Comment:
I have a feeling that this is overall just wrong.  No?

Reply:
Could we try `moduletestutil.MakeTestEncodingConfig(ica.AppModuleBasic)`?

ref: https://github.com/cosmos/cosmos-sdk/blob/v0.50.0-alpha.1/types/module/testutil/codec.go#L24

Reply:
> Could we try moduletestutil.MakeTestEncodingConfig(ica.AppModuleBasic{})?

Yea that should work